components/more-less/more-less.js

```diff
@@ -234,13 +237,27 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 		this.expanded = true;
 	}
 
-	__focusIn() {
+	__focusIn(e) {
 		if (this.inactive || this.expanded) {
 			return;
 		}
 
-		this.__expand();
-		this.__autoExpanded = true;
+		const target = e.composedPath()[0];
+
+		if (isSafari) {
+			target.scrollIntoViewIfNeeded();
+		}
+
+		if (this.__content.scrollTop) {
+			this.__content.scrollTo({ top: 0, behavior: 'instant' });
+			this.__expand();
+			if (reduceMotion) {
+				target.scrollIntoView({ behavior: 'instant' });
+			} else {
+				setTimeout(() => target.scrollIntoView({ behavior: 'smooth' }, 100));
+			}
```

Comment:
...then we can expand, then make sure the target is in view.


Amazingly, `smooth` scroll is still used with reduced motion, so we need to disable it manually. The delay ensures that the smooth scroll doesn't miscalculate as the expand is still happening, and roughly times them together for less jumpiness.